core/arch/arm/tee/entry_std.c

```diff
@@ -101,6 +101,7 @@ static TEE_Result set_tmem_param(const struct optee_msg_param_tmem *tmem,
 static TEE_Result set_rmem_param(const struct optee_msg_param_rmem *rmem,
 				 struct param_mem *mem)
 {
+	size_t req_size = 0;
```

Comment:
I started out writing the majority of the patches, but in some cases Jens tweaked them. This might be one of them. I'll have a check and will update-accordingly when I force/squash push.

Reply:
Fixed on patch just pushed.